packages/react-ui-core/src/Dropdown/Dropdown.js

```diff
@@ -53,6 +53,16 @@ export default class Dropdown extends Component {
     return parseArgs(anchorField, AnchorButton)
   }
 
+  renderChildren() {
+    const {
+      children,
+    } = this.props
+
+    return cloneElement(children, {
```

Comment:
What if children is an array of children?

Reply:
Good point. I'll use `React.children` and iterate over them if that is the case

Reply:
You probably want: 
```
const childrenWithProps = React.Children.map(children, child =>
  React.cloneElement(child, { toggleVisibility: this.toggleVisibility }))

return childrenWithProps
```

Reply:
haha I literally just wrote exactly that 😉